projects/template.md

```diff
@@ -67,10 +73,40 @@ After the project is accepted, the Network Steward will pay XXX PKT
 
 ### Milestone 1
 
-> At this point, a user will be able to ...
+#### Success criteria
+> **Instructions**: This is where you write success criteria which will be used for verifying that
+milestone 1 was delivered correctly.
 
-> After a report is delivered on this milestone and it is approved by the NS, the NS will send XXX PKT.
+#### Payout
+After a report is delivered on this milestone and it is approved by the NS, the NS will send **TODO** PKT.
 
 ### Milestone 2
 
-...
\ No newline at end of file
+...
+
+## Disclosure
+I hereby submit this application in good faith and I attest that I have made no effort, nor do I
+intend to make effort, influence the Network Steward to accept this or any other project I have
+submitted.
```

Comment:
We should maybe add a little more to this statement. It would be nice if there was also an attestation to the effect that, if the proposal is accepted, the applicant won't use their work on the project itself to do something detrimental to the pkt network. Some hypothetical bad-faith work on reasonable projects might be:
1. Writing documentation (e.g. whitepaper) which is misleading in a way that omits problems or over-states successes, in an effort to drive up the value of pkt.
2. Writing an application (wallet, exchange, etc.) with an intentional security vulnerability that allows the applicant to steal pkt (ok, this is probably illegal anyway, it's mostly to protect the NS if an applicant does something unexpectedly malicious and tries to play the "there's no rule that dogs can't play basketball" defense).
3. Writing an application (price ticker etc.) that has an intentional bug or design limitation which causes it to produce incorrect results, in a manner which would e.g. give incorrect information about pkt prices in an effort to manipulate the market. (See e.g. the 2015 underhanded c challenge "faking fissile material" for an example of how a "bug" can produce very specific and misleading incorrect results.)

Reply:
I'm worried about "contract scope creep" here:

1. Someone either doesn't know what they're doing and they're just excited, or they do know and they will have no problem checking the box anyway (and will claim that it's the truth and is not misleading... then if they're completely discredited they will then fall back on "I didn't know")
2. Is a very clear case where a person who was going to do that would check the box anyway and it doesn't provide us any legal protection we didn't already have - either we had no idea they planned to put the bug and we are innocent or we did know and we're guilty, regardless of the box.
3. This is kind of the same question as 1, anyone who did this would claim they had no idea and the presence of a checkbox would not dissuade them.

So in all of these cases, the crook has plausible deniability and adding the checkboxes really only serves to increase effort and mental load on innocent people. We must not underestimate the anxiety which can be created, in honest people with no criminal intent, by things such as:  "[ ] I attest that I have not committed any violation of the Internal Revenue Code in the past 7 years"

What we could do is ask whether the project result will: "Present information which might lead to price speculation" and if the applicant says yes, they must add a paragraph explaining the controls which will be in place to prevent people being mislead by the information.

WDYT?

Reply:
FYI I've added the "Present information which might lead to price speculation" checkbox in the most recent push.